scale/docs/rest/v6/system.rst

```diff
@@ -0,0 +1,324 @@
+
+.. _rest_system:
+
+v6 System Services
+===============
+
+These services provide access to general system information.
+
+.. _rest_v6_system_status:
+
++-------------------------------------------------------------------------------------------------------------------------------+
+| **Get Scheduler Status**                                                                                                      |
++===============================================================================================================================+
+| Returns the current status of the scheduler, including information about nodes and running jobs.                              |
++-------------------------------------------------------------------------------------------------------------------------------+
+| **GET** /v6/status/                                                                                                              |
```

Comment:
Missing the right side | chars for this section. Should move the JSON responses out of the table (like in the new v6 batch API).